soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -11097,7 +11097,7 @@ s16 func_8084ABD8(GlobalContext* globalCtx, Player* this, s32 arg2, s16 arg3) {
     else {
         temp1 = (this->stateFlags1 & PLAYER_STATE1_23) ? 3500 : 14000;
         temp3 = ((sControlInput->rel.stick_y >= 0) ? 1 : -1) *
-            (s32)((1.0f - Math_CosS(sControlInput->rel.stick_y * 200)) * 1500.0f);
+            (s32)((1.0f - Math_CosS(sControlInput->rel.stick_y * 200)) * (CVar_GetS32("gUninvertAim", 0) != 0 ? -1500.0f : 1500.0f));
```

Comment:
```suggestion
            (s32)((1.0f - Math_CosS(sControlInput->rel.stick_y * 200)) * 1500.0f * (CVar_GetS32("gUninvertAim", 0) ? -1 : 1));
```

Reply:
This makes it not function. Please test code before suggesting changes.

Reply:
Why is it wrong?

Reply:
Ah crap, misread and replicated the code wrong, I thought all you'd done was replace the 1500 values with 1, which would break the code.

My bad.